code/modules/clothing/suits/_lasertag_components.dm

```diff
@@ -0,0 +1,18 @@
+/datum/component/lasertag
+	///What team the mob that this component is attached to is part of. This should be all lowercase and either a color or "neutral"
+	var/team_color = "neutral"
+	///Anything that makes a human a valid lasertag target should be added to this
+	var/list/lasertag_granters = list()
+
+/datum/component/lasertag/Initialize(...)
+	. = ..()
+	var/mob/living/carbon/human/H = parent
+	if (!H)
+		return COMPONENT_INCOMPATIBLE
```

Comment:
1. You don't need to call parent, doesn't do anything

2. This check will never fail, as you're not checking if parent is a human. 

```dm
if(!ishuman(parent))
	return COMPONENT_INCOMPATIBLE
```